packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx

```diff
@@ -183,11 +185,12 @@ export const ChatScreen = (props: ChatScreenProps): JSX.Element => {
               styles={sendBoxStyles}
             />
 
-            <FileUploadButton
+            {/* REMOVE THE FILE SHARING BUTTON FOR 1.1.1-BETA.1 RELEASE -- DO NOT COMMIT THIS TO THE MAIN BRANCH */}
```

Comment:
I would have tried to make this even smaller.

```ts
const willNeverBeTrue = process.env['DO_NOT_SET_THIS'] === 'True';

...

{ willNeverBeTrue && <FileUploadButton ...>}
```

Then you don't need to remove any imports etc.

Reply:
Oh yah much simpler, updated with a const `const SHOW_FILE_SHARING_BUTTON = false;`

Reply:
Heh, I have a healthy respect for compiler optimizations. So if I want the compiler to keep some code, I tie it to something unguessable like environment variable :D